src/targets/google-tag-manager/__tests__/google-tag-manager.test.js

```diff
@@ -1,8 +1,11 @@
 const { GoogleTagManager } = require('../google-tag-manager');
 
-beforeEach(() => { window.dataLayer = undefined; });
+beforeEach(() => {
+  window.dataLayer = undefined;
+  window.iAmADataLayer = undefined;
+});
 
-describe('GoogleTagManager(events)', () => {
+describe('GoogleTagManager({...settings})(events)', () => {
```

Comment:
I think `options` might be a better name for the signature, as it isn't required.